src/helpers/setCallbacksFn.js

```diff
@@ -0,0 +1,15 @@
+import { validateCallbackFn } from './validateCallbackFn'
+
+/**
+ *
+ * @param {Array} callbacks
+ *
+ * @return {Array} callbacks
+ */
+export function setCallbacksFn (callbacks) {
```

Comment:
What is the idea behind this function? It's not used anywhere. 

Also, if it should exist, please add a test case. If possible, adding this (with test case) could also be a separate PR.